src/zenml/__init__.py

```diff
@@ -27,6 +27,14 @@
 
 init_logging()
 
+# Initialize source context
```

Comment:
Is this really something that needs to be in the public top level `__init__`? AFAIK this isn't a user-facing feature right?

Reply:
Fixed it and moved it to the analytics module.